core/primitives-core/src/version.rs

```diff
@@ -147,6 +147,7 @@ pub enum ProtocolFeature {
     EthImplicitAccounts,
     /// Enables yield execution which is introduced in <https://github.com/near/NEPs/pull/519>
     YieldExecution,
+    FixMinStakeRatio,
```

Comment:
nit: Can you add a short comment and maybe a link if there is a good full description of the problem somewhere?

Reply:
Yeah, done.